packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -37,6 +37,8 @@ export interface RollupLwcOptions {
     enableScopedSlots?: boolean;
     /** The configuration to pass to `@lwc/compiler` to disable synthetic shadow support */
     disableSyntheticShadowSupport?: boolean;
+    /** Boolean to determine if LWS is on or off */
+    lws?: boolean;
```

Comment:
Technically there is not need to add the `lws` param here unless you need it for testing. `@lwc/rollup-plugin` is not used on platform.

Reply:
If you decide to keep the option, please update the `README.md` of this package.

Reply:
I think we (LWS) are more or less opinionated  and just looking for the proper places to put the option. If you all (LWC) tell us where it should go we're more than happy to make it so. We can remove from rollup  for simplicity (it can always come later if needed for testing).

Reply:
If nobody's using it, I'd say YAGNI and let's remove it from the `rollup-plugin`.

Reply:
Perfect :)